src/transformers/models/bloom/configuration_bloom.py

```diff
@@ -13,7 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """ Bloom configuration"""
+from collections import OrderedDict
+from typing import Any, List, Mapping, Optional
+
+from transformers import PreTrainedTokenizer, TensorType, is_torch_available
```

Comment:
I would use strings for type annotations and something like:
```
from typing import TYPE_CHECKING

if TYPE_CHECKING:
   from transformers import ...
```
That way you don't import classes that are not actually being used.